src/forms/forms.response.ts

```diff
@@ -97,8 +97,23 @@ class Question {
     @ApiProperty({ example: null })
     optionGroup: OptionGroup;
 
-    @ApiProperty({ isArray: true })
-    responses: Response;
+    @ApiProperty()
+    createdAt: Date;
+
+    @ApiProperty()
+    updatedAt: Date;
+}
+
+class SubQuestion extends BaseQuestion {
+    // No need for parentQuestionId in SubQuestion
+    // @ApiProperty()
+    // parentQuestionId: number;
```

Comment:
```suggestion
```

Reply:
sure, no problem! will finish with the changes and update PR later today